docs/source/API/extension_dev.rst

```diff
@@ -1,6 +1,6 @@
 .. _develop-extensions:
 
-Create an Extension
+Create an extension
```

Comment:
I prefer the original but I also realize we are not consistent. For v1.0 we will update all headers in the documentation.